docs/current_docs/api/snippets/custom-types/typescript/index.ts

```diff
@@ -19,30 +19,31 @@ class Account {
   }
 }
 
-@object()
-class Organization {
-  @func()
+/**
```

Comment:
@TomChv should the Python and Go snippets also be updated?

Reply:
No, this change basically allows TypeScript to define simple objects in other ways that feel more natural to TypeScript developers that want to use them. So it's a "multiple ways of doing the same thing" kind of thing.

If a user wants to use a "simple object", then it can't have functions. It's just fields, and they're all exposed to the Dagger API. For this use case that's probably what the user wants, the problem is if they then want to add a function they'd have to change the type to a class, but I think that would be expected for a TypeScript user as it's not Dagger specific.